MOBILE_API.xml

```diff
@@ -3699,7 +3688,428 @@
+        </param>
+
+        <param name="allowAppConsumers" type="Boolean" mandatory="false" defvalue="false">
+            <description> If true, app service consumers beyond the IVI system will be able to access this service. If false, only the IVI system will be able consume the service. If not provided, it is assumed to be false. </description>
+        </param>
+
+        <param name="rpcSpecVersion" type="SyncMsgVersion" mandatory="false">
+            <description> This is the max RPC Spec version the app service understands. This is important during the RPC passthrough functionality. If not included, it is assumed the max version of the module is acceptable. </description>
+        </param>
+
+        <param name="handledRPCs" type="Integer" array="true" mandatory="false">
+            <description> This field contains the Function IDs for the RPCs that this service intends to handle correctly. This means the service will provide meaningful responses. </description>
+        </param>
+
+        <param name="mediaServiceManifest" type="MediaServiceManifest" mandatory="false"/>
+        <param name="weatherServiceManifest" type="WeatherServiceManifest" mandatory="false"/>
+        <param name="navigationServiceManifest" type="NavigationServiceManifest" mandatory="false"/>
+    </struct>
+
+    <struct name="AppServiceRecord" since="5.1">
+        <description> This is the record of an app service publisher that the module has. It should contain the most up to date information including the service's active state</description>
+
+        <param name="serviceID" type="String" mandatory="true">
+            <description> A unique ID tied to this specific service record. The ID is supplied by the module that services publish themselves. </description>
+        </param>
+
+        <param name="serviceManifest" type="AppServiceManifest" mandatory="true">
+            <description> Manifest for the service that this record is for.</description>
+        </param>
+
+        <param name="servicePublished" type="Boolean" mandatory="true">
+            <description> If true, the service is published and available. If false, the service has likely just been unpublished, and should be considered unavailable.</description>
+        </param>
+
+        <param name="serviceActive" type="Boolean" mandatory="true">
+            <description> If true, the service is the active primary service of the supplied service type. It will receive all potential RPCs that are passed through to that service type. If false, it is not the primary service of the supplied type. See servicePublished for its availability. </description>
+        </param>
+    </struct>
+
+    <struct name="AppServiceData" since="5.1">
+        <description> Contains all the current data of the app service. The serviceType will link to which of the service data objects are included in this object. (eg if service type equals MEDIA, the mediaServiceData param should be included.</description>
```

Comment:
_object. (eg if service type equals MEDIA, the mediaServiceData param should be included._
The text above has some formatting issues. It should be:
_object (e.g. if the service type is MEDIA, the mediaServiceData param should be included)._

Reply:
Fixed in https://github.com/smartdevicelink/rpc_spec/pull/149/commits/06a3394c636aa24d9b880a352c67c94700476e0e